server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -50,14 +55,9 @@ import { AmazonQTokenServiceManager } from '../../shared/amazonQServiceManager/A
 
 type ChatHandlers = Omit<
     LspHandlers<Chat>,
-    | 'openTab'
-    | 'sendChatUpdate'
-    | 'onFileClicked'
-    | 'onInlineChatPrompt'
-    | 'sendContextCommands'
-    | 'onCreatePrompt'
-    | 'onListConversations'
-    | 'onConversationClick'
+    'openTab' | 'sendChatUpdate' | 'onFileClicked' | 'onInlineChatPrompt' | 'sendContextCommands' | 'onCreatePrompt'
+    // | 'onListConversations'
```

Comment:
Why are these commented?

Reply:
I'll revert code in `language-server` - used it for testing.